src/pages/TokenDetails/index.tsx

```diff
@@ -23,50 +29,9 @@ import { useOnGlobalChainSwitch } from 'hooks/useGlobalChainSwitch'
 import { useAtomValue } from 'jotai/utils'
 import { useTokenFromQuery } from 'lib/hooks/useCurrency'
 import useCurrencyBalance, { useTokenBalance } from 'lib/hooks/useCurrencyBalance'
-import { useCallback, useState } from 'react'
+import { useCallback, useState, useTransition } from 'react'
 import { ArrowLeft } from 'react-feather'
 import { useNavigate, useParams } from 'react-router-dom'
-import styled from 'styled-components/macro'
-
-const Hr = styled.hr`
```

Comment:
These are included in the non-lazy-loaded skeleton, to avoid eagerly loading TokenDetails (which is meant to be lazy-loaded).